lua/nvim-possession/init.lua

```diff
@@ -117,6 +117,29 @@ M.setup = function(user_opts)
 		end
 	end
 
+	---rename selected session
+	---@param selected string
+	M.rename_selected = function(selected)
+        local session_path = user_config.sessions.sessions_path
+        local old_name = selected[1]
+        local old_file_path = session_path .. old_name
+
+        -- Prompt user for the new name
+        local new_name = vim.fn.input("Enter new name for the session: ", old_name)
+        
+        if new_name and new_name ~= "" then
+            local new_file_path = session_path .. new_name
+
+            -- Rename the file
+            os.rename(old_file_path, new_file_path)
+            
+            print("Session renamed from " .. old_name .. " to " .. new_name)
+        else
+            print("New name cannot be empty.")
```

Comment:
Could you please replace the `print` statements with `vim.notify`? Something along the lines of
```lua
vim.notify(
  "New name cannot be empty",
  vim.log.levels.WARN,
  { title = notification_title }
)
```
and likewise for the other print statements?